ubuntuHomestead.py

```diff
@@ -214,10 +217,11 @@ def install(program_name, common_name):
 python_program.close()
 
 if md5(file_name) == FILE_HASH:
-	os.system('wget $(python {})'.format(file_name))
+    os.system('wget $(python {})'.format(file_name))
 else:
-	print('Python script integrity compromised. Exiting now')
-	exit(1)
+    print('Python script integrity compromised. Exiting now')
+    # TODO: Update the hash at the top of this script.
```

Comment:
I didn't want to touch your hash code.

Reply:
The hash is actually for the program that I use to get the download link for vagrant. There is no need to comment out the  exit call